testsuite/httpx/__init__.py

```diff
@@ -53,6 +53,10 @@ def close(self) -> None:
             file.close()
         self.files = []
 
+    def add_retry_code(self, code):
+        """Add a new retry code to"""
+        self.RETRY_CODES.add(code)
```

Comment:
`RETRY_CODES` is static so this will add it to every Client that exists and will so in the future